app/uk/gov/hmrc/agentclientmandate/config/ScheduledJobStarter.scala

```diff
@@ -61,17 +63,18 @@ class DefaultScheduledJobStarter @Inject()(val app: Application,
     }
   }
 
-  cancellables = scheduledJobs.map { job =>
-    scheduler(app).scheduleWithFixedDelay(job.initialDelay, job.interval)(new RunnableTask(job))
-  }
+  cancellables = scheduledJobs.map(job => scheduler(app).scheduleWithFixedDelay(job.initialDelay, job.interval)(new RunnableTask(job)(ec)))
```

Comment:
```suggestion
  cancellables = scheduledJobs.map(job => scheduler(app).scheduleWithFixedDelay(job.initialDelay, job.interval)(new RunnableTask(job)))
```
Compiles (for me at least) without passing explicitly.